src/content/docs/pl/getting-started.mdx

```diff
@@ -2,108 +2,63 @@
-</div>
-
-## Rozpocznij swój pierwszy projekt
-
-Stwórz nowy projekt Astro lokalnie dzięki naszemu pomocnemu kreatorowi `create astro` CLI!
-
-<PackageManagerTabs>
-  <Fragment slot="npm">
-  ```shell
-  # utwórz nowy projekt z npm
-  npm create astro@latest
-  ```
-  </Fragment>
-  <Fragment slot="pnpm">
-  ```shell
-  # utwórz nowy projekt z pnpm
-  pnpm create astro@latest
-  ```
-  </Fragment>
-  <Fragment slot="yarn">
-  ```shell
-  # utwórz nowy projekt z yarn
-  yarn create astro
-  ```
-  </Fragment>
-</PackageManagerTabs>
-
-Nasz [Przewodnik po instalacji](/pl/install/auto/) posiada pełne instrukcje krok-po-kroku dotyczące instalacji Astro za pomocą naszego kreatora CLI, tworzenia nowego projektu z istniejącego repozytorium Astro GitHub oraz ręcznej instalacji Astro.
-
-
-## Naucz się Astro
-
-Zobacz przykłady niektórych kluczowych pojęć i wzorców witryny Astro!
-
-📚 [Dodaj swoją pierwszą stronę](/pl/basics/astro-pages/) do swojej witryny.
-
-📚 Dowiedz się więcej o [strukturze projektu](/pl/basics/project-structure/) w Astro.
-
-📚 Dowiedz się więcej o [routingu opartego na plikach](/pl/guides/routing/).
-
-*... naszą pełną dokumentację API znajdziesz w zakładce **Referencje**.*
-
-
-## Rozbuduj swój projekt Astro
-
-🧰 Zacznij swój następny projekt od [gotowego motywu](https://astro.build/themes/)
-
-🧰 Dostosuj swoją stronę za pomocą oficjalnych i społecznościowych [wtyczek i komponentów](https://astro.build/integrations/).
-
-🧰 Zainspiruj się, odwiedzając naszą [galerię stron Astro](https://astro.build/showcase/).
-
-*... zobacz nasz [przewodnik po korzystaniu z integracji](/pl/guides/integrations-guide/)*.
-
-
-
-## Dołącz do naszej społeczności
-
-Dołącz do nas na [Discordzie Astro](https://astro.build/chat), aby podzielić się z i uzyskać pomoc od aktywnej, przyjaznej społeczności!
-
-💬 Przywitaj się na naszym kanale `#general`!
-
-💬 Zadaj pytanie naszemu Zespołowi Wsparcia na kanale `#support`!
-
-💬 Podziel się tym, nad czym pracujesz na kanale `#showcase`!
-
-
-## Co nowego?
-
-[Blog Astro](https://astro.build/blog/)
-
-[Lista zmian w Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
-
-## Wesprzyj nas
-
-Ta dokumentacja jest dostarczana do ciebie przez wszystkich tych pomocnych ludzi. [Dołącz do nas na GitHubie!](https://github.com/withastro/docs)
-
-<ContributorList githubRepo="withastro/docs" />
+import { CardGrid } from '@astrojs/starlight/components'
+import Card from '~/components/Landing/Card.astro'
+import ListCard from '~/components/Landing/ListCard.astro'
+import SplitCard from '~/components/Landing/SplitCard.astro'
+import Discord from '~/components/Landing/Discord.astro'
+
+<CardGrid>
+  <Card title="Co stworzysz z Astro?" icon="laptop">
+    Eksploruj [startowe arkusze Astro](https://astro.build/themes/) aby znaleźć blogi, portfolia, dokumentacje, strony startowe, SaaS, marketing, strony ecommerce oraz więcej!
```

Comment:
What do you think about 'startowe motywy Astro'?

Reply:
Oh yeah, that sounds good!